src/picknik_ur_multi_arm_config/config/control/picknik_multi_ur.ros2_control.yaml

```diff
@@ -0,0 +1,339 @@
+controller_manager:
+  ros__parameters:
+    update_rate: 600  # Hz
+    joint_state_broadcaster:
+      type: joint_state_broadcaster/JointStateBroadcaster
+    first_io_and_status_controller:
+      type: ur_controllers/GPIOController
+    second_io_and_status_controller:
+      type: ur_controllers/GPIOController
+    third_io_and_status_controller:
+      type: ur_controllers/GPIOController
+    fourth_io_and_status_controller:
+      type: ur_controllers/GPIOController
+    multi_arm_joint_trajectory_controller:
+      type: joint_trajectory_controller/JointTrajectoryController
+    first_robotiq_gripper_controller:
+      type: position_controllers/GripperActionController
+    first_robotiq_activation_controller:
+      type: robotiq_controllers/RobotiqActivationController
+    second_robotiq_gripper_controller:
+      type: position_controllers/GripperActionController
+    second_robotiq_activation_controller:
+      type: joint_trajectory_controller/JointTrajectoryController
+    third_robotiq_gripper_controller:
+      type: position_controllers/GripperActionController
+    third_robotiq_activation_controller:
+      type: joint_trajectory_controller/JointTrajectoryController
+    fourth_robotiq_gripper_controller:
+      type: position_controllers/GripperActionController
+    fourth_robotiq_activation_controller:
+      type: joint_trajectory_controller/JointTrajectoryController
+    first_servo_controller:
+      type: joint_trajectory_controller/JointTrajectoryController
+    second_servo_controller:
+      type: position_controllers/JointGroupPositionController
```

Comment:
Why are there two different controllers here? I noticed the same in the 2 arm version but they seem to both work interchangeably

Reply:
I removed the servo controllers entirely